benchmarks/src/main/scala/scalaz/data/StringBalancedConcatFoldBenchmark.scala

```diff
@@ -0,0 +1,50 @@
+package scalaz
```

Comment:
Could be worth including Cats for reference like the `IO` benchmarks.

Reply:
cats doesn't have a Cord; I will include Scalaz 7's Cord though.

Reply:
It has `Show` though. The benchmark could compare `Show` vs `Show` (7) vs `Debug` (8).

Reply:
Good point.